src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+body,
+ul {
+  box-sizing: border-box;
+  margin: unset;
+  padding: unset;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  height: 60px;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 10px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 10px;
+}
+
+.nav__link {
+  position: relative;
+  color: #000;
+  text-decoration: none;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
```

Comment:
Pleased don't use repeated styles. You already have this styles in:
![image](https://user-images.githubusercontent.com/54936044/229482394-35f6a80c-4cf5-4a04-923c-5c02e98b08b8.png)